tests/integration/test_throttling/test.py

```diff
@@ -114,7 +114,7 @@ def node_update_config(mode, setting, value=None):
 
 
 def assert_took(took, should_took):
-    assert took >= should_took[0] * 0.9 and took < should_took[1]
```

Comment:
@antonio2368 thanks! How about also put a comment why this drift is possible? I guess that this should be possible only for server limits, since they can do some reading/writing in background, which will throttle that IO instead of IO for query.

Reply:
Yep, looks good, thanks!

(P.S. I cannot use approve button since now I don't have it, apparently something had been changed either in project settings or in the github)